### ISSUE TITLE
About C++ interface

### ISSUE THREAD

**Issue by PuYuuu** [CommonRoad/commonroad-io#9]
I like commroad very much, but I would rather use c++ than python to implement my algorithm. Does CommonRoad-IO have C++ interface ? Thanks for your work !😀

**Comment by smaierhofer** [comment]
Currently, there exist no publicly available C++ interface.
We have an internal implementation covering parts of commonroad-io, e.g., file-reading, road network, obstacles (but no visualization).
We probably release the tool next year (maybe already this year in a basic draft version).

**Comment by PuYuuu** [comment]
-Got it, thank you!

**Comment by PuYuuu** [comment]
Hi @smaierhofer , Is the C++ interface currently available? I would like to do visualization based on the open source Foxglove Studio.

**Comment by smaierhofer** [comment]
not, yet. 
We're planning to release it by end of September.

**Comment by PuYuuu** [comment]
> not, yet. We're planning to release it by end of September.

Once again, may I ask if the C++ interface of commonroad-io has been released?

**Comment by smaierhofer** [comment]
yes, it is available here: https://github.com/CommonRoad/environment-model.
However, it is still a beta version.

**Comment by PuYuuu** [comment]
Got it, thanks. BTW, I would like to know whether it is possible to add custom dynamic obstacles and specify their trajectory or lane in the commonroad-scenario-designer tool? Is there any detailed documentation? I can't find it on the official website https://commonroad.in.tum.de/tools/scenario-designer.

**Comment by smaierhofer** [comment]
No, this is currently not possible.
